scripts/ci/docker-compose/backend-mysql.yml

```diff
@@ -36,6 +38,12 @@ services:
       - ../mysql/conf.d:/etc/mysql/conf.d:ro
       - /dev/urandom:/dev/random   # Required to get non-blocking entropy source
       - mysql-db-volume:/var/lib/mysql
+    healthcheck:
+      test: ["CMD", "mysql", "-h", "localhost", "-P", "3306", "-u", "root", "-e", "SELECT 1"]
+      interval: 10s
+      timeout: 10s
+      retries: 120
+
```

Comment:
Is there any specific reason why `restart: always` is not added for `mysql` and `postgres`?

Reply:
Good point. I missed it. Added.